tests/unit/ResultsListItem.spec.js

```diff
@@ -2,7 +2,7 @@ import { mount } from '@vue/test-utils'
 import ResultsListItem from '@/components/ResultsListItem.vue'
 
 describe('ResultsListItem', () => {
-  it('renders ResultsListItem component', () => {
+  it.skip('renders ResultsListItem component', () => {
```

Comment:
Thanks for pointing out you're skipping this in the PR. I'll update the tests to incorperate Vuex today.